File: src/CSPAF/CSPCore/CSPViewControllers/CSPPopToRootController.cs
using System;
using System.Collections.Generic;

using CloudSuitePortal;
using CloudSuitePortal.Services;

class CSPPopToRootController
{
    public void run(Dictionary<string, object> Params) {
        CSPBrowser.Application.WindowController.NavigationController.PopToRootViewController();
    }
}

File: src/CSPAF/CSPCore/CSPViewControllers/CSPPopToRootController.java
package CSPAF.CSPCore.CSPViewControllers;

import java.util.*;

import com.infor.cloudsuite.portal.core.services.*;

class CSPPopToRootController
{
    public void run(HashMap<String, Object> params) {
        CSPBrowser.getApplication().getWindowController().getNavigationController().popToRootViewController();
    }
}